.changeset/sharp-ravens-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vue-storefront/eslint-config": minor
+---
+
+**[CHANGED]** some rules to loose up linting a bit (non-breaking)
```

Comment:
Can we add precise info what rules has changed?

Reply:
sure, done 902ff20a7584a7677f9306c0b6890e4236b6dded